### PR TITLE
Run python unit tests on every platform + fix all string operators

### PR DESCRIPTION
**Pull request by xadupre**
Signed-off-by: xavier dupré <xavier.dupre@gmail.com>

### CODE REVIEW

File: test/test_gpt2tok.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
     enable_custom_op,
     PyCustomOpDef,
     expand_onnx_inputs,
-    get_library_path as _get_library_path)
+    get_library_path as _get_library_path,
```
Comment (xadupre) [inline test/test_gpt2tok.py:13]: remove

File: includes/onnxruntime/onnxruntime_cxx_api.h [hunk 2]
```diff
@@ -599,6 +599,7 @@ struct CustomOpApi {
   OrtValue* KernelContext_GetOutput(OrtKernelContext* context, _In_ size_t index, _In_ const int64_t* dim_values, size_t dim_count);
 
   void ThrowOnError(OrtStatus* result);
+  const OrtApi& Api() { return api_; }
```
Comment (wenbingl) [inline includes/onnxruntime/onnxruntime_cxx_api.h:602]: These headers are directly copied for ORT release, which suppose not be changed.

Comment (xadupre) [reply includes/onnxruntime/onnxruntime_cxx_api.h:602]: Retrieving input string tensors is not easy. The function used for the other types fail on Linux, there is no KernelContext_* methods to do so. The only way is go handle them directly from the API. I need to raise an issue on onnxruntime. This is a temporary solution.

File: test/test_gpt2tok.py (hunk 1 above)
Comment (xadupre) [reply test/test_gpt2tok.py]: done

File: includes/onnxruntime/onnxruntime_cxx_api.h (hunk 2 above)
Comment (xadupre) [reply includes/onnxruntime/onnxruntime_cxx_api.h]: I removed it by using a trick which should be replaced by something cleaner when the C API is updated.

File: includes/ocos.h [hunk 3]
```diff
@@ -6,6 +6,7 @@
 #define ORT_API_MANUAL_INIT
 #define EXCLUDE_REFERENCE_TO_ORT_DLL
 #include "onnxruntime_cxx_api.h"
+#include "onnxruntime_cxx_inline.h"
```
Comment (wenbingl) [inline includes/ocos.h:9]: the *_inline.h is included at the footer of *_cxx_api.h, this change is unnecessary.

File: includes/onnxruntime/onnxruntime_cxx_inline.h [hunk 4]
```diff
@@ -7,6 +7,8 @@
 // These are the inline implementations of the C++ header APIs. They're in this separate file as to not clutter
 // the main C++ file with implementation details.
 
+#pragma once
```
Comment (wenbingl) [inline includes/onnxruntime/onnxruntime_cxx_inline.h:10]: not needed.

File: includes/ocos.h (hunk 3 above)
Comment (xadupre) [reply includes/ocos.h]: done

File: includes/onnxruntime/onnxruntime_cxx_inline.h (hunk 4 above)
Comment (xadupre) [reply includes/onnxruntime/onnxruntime_cxx_inline.h]: done

File: ci_build/azure-pipelines/mshost.yaml [hunk 5]
```diff
@@ -31,26 +35,26 @@ jobs:
...
 
-  - script: |
-      python -m pip install -r requirements-dev.txt
+  - script: python -m pip install -r requirements-dev.txt
     displayName: Install requirements-dev.txt
 
-  - script: |
-      # FIXME: need check the CI environment for the failure.
-      # python -m pytest test
+  - script: python -m pytest test --verbose --verbose
```
Comment (wenbingl) [inline ci_build/azure-pipelines/mshost.yaml:44]: two verbose?

**Review by wenbingl — APPROVED** [review]

File: includes/onnxruntime/onnxruntime_cxx_inline.h (hunk 4 above)
Comment (xadupre) [reply includes/onnxruntime/onnxruntime_cxx_inline.h]: I prefer to keep it that way. When python crashes, there is no way to tell which test failed unless we keep that verbosity.

**Pull request merged**